Sources/REventLogger/Networking/REventLoggerSender.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+protocol REventLoggerSendable {
+    func sendEvents(_ apiUrl: String, events: [REvent], onCompletion: @escaping (Result<Data, Error>) -> Void)
```

Comment:
We can send the decoded model in the success case instead of data.

Reply:
Since we donot get a body in the success response , so using a decoded model is not required here